indy_client/test/cli/test_pool_upgrade.py

```diff
@@ -1,137 +0,0 @@
-from copy import copy
-
-import pytest
-
-from indy_node.test import waits
-from stp_core.loop.eventually import eventually
-from plenum.common.constants import VERSION
-from indy_common.constants import ACTION, CANCEL, JUSTIFICATION
-from indy_node.test.upgrade.helper import checkUpgradeScheduled, \
-    checkNoUpgradeScheduled
-from indy_node.test.upgrade.conftest import validUpgrade, validUpgradeExpForceFalse, validUpgradeExpForceTrue
-
-
-def send_upgrade_cmd(do, expect, upgrade_data):
```

Comment:
This tests need to be migrated to SDK and moved to indy-node folder

Reply:
I think we have every tests case duplicated in
indy-node/indy_node/test/upgrade
Please, tell me, if we need to integrate this in node